data/json/faults/faults_electronic.json

```diff
@@ -0,0 +1,71 @@
+[
+  {
+    "type": "fault",
+    "id": "fault_electronic_blown_fuse",
+    "fault_type": "shorted",
+    "name": { "str": "Blown fuse" },
+    "description": "Required for closing a protected electronic circuit.",
+    "item_prefix": "shorted",
+    "flags": [ "ITEM_BROKEN" ]
+  },
+  {
+    "type": "fault_fix",
+    "id": "mend_fault_electronic_blown_fuse",
+    "name": "Replace blown fuse",
+    "success_msg": "You replace the blown fuse of the %s.",
+    "time": "5 m",
+    "faults_removed": [ "fault_electronic_blown_fuse" ],
+    "skills": { "electronics": 3 },
+    "requirements": [ { "qualities": [ { "id": "SCREW", "level": 1 } ], "components": [ [ [ "e_scrap", 1 ], [ "copper_wire", 1 ] ] ] } ]
+  },
+  {
+    "type": "fault",
+    "id": "fault_electronic_blown_capacitor",
+    "fault_type": "shorted",
+    "name": { "str": "Blown capacitor" },
+    "description": "Required for storing charge in an electronic circuit.",
+    "item_prefix": "shorted",
+    "flags": [ "ITEM_BROKEN" ]
+  },
+  {
+    "type": "fault_fix",
+    "id": "mend_fault_electronic_blown_capacitor",
+    "name": "Replace blown fuse",
+    "success_msg": "You replace the blow capacitor of the %s.",
+    "time": "10 m",
+    "faults_removed": [ "fault_electronic_blown_capacitor" ],
+    "skills": { "electronics": 4 },
+    "requirements": [
+      {
+        "qualities": [ { "id": "SCREW", "level": 1 } ],
+        "tools": [ [ "soldering_iron" ], [ "soldering_iron_portable" ] ],
+        "components": [ [ [ "e_scrap", 1 ] ] ]
+      }
+    ]
+  },
+  {
+    "type": "fault",
+    "id": "fault_electronic_shorted_circuit",
+    "fault_type": "shorted",
+    "name": { "str": "Shorted electronic circuit" },
+    "description": "An electronic circuit has been destroyed by overvoltage.",
+    "item_prefix": "shorted",
+    "flags": [ "ITEM_BROKEN" ]
+  },
+  {
+    "type": "fault_fix",
+    "id": "mend_fault_electronic_shorted_circuit",
+    "name": "Replace shorted circuit",
+    "success_msg": "You replace the shorted circuit of the %s.",
+    "time": "70 m",
+    "faults_removed": [ "fault_electronic_shorted_circuit" ],
+    "skills": { "electronics": 5 },
+    "requirements": [
+      {
+        "qualities": [ { "id": "SCREW", "level": 1 } ],
+        "tools": [ [ "soldering_iron" ], [ "soldering_iron_portable" ] ],
+        "components": [ [ [ "circuit", 1 ] ] ]
```

Comment:
I'm not sure it makes sense to be able to replace a shorted circuit board in an arbitrary electronic device with a random other one you found. Especially for e.g. e-ink PCs, smartphones, etc—replacing these is a highly specialized task that requires specific tools and exact matches for components.

Reply:
Sure, but just like e-inks, laptops, and smartphone items, the circuit item is an interchangeable abstraction. 

Maybe the cost should be increased to a number of circuits, representing testing until you find a matching one? Thoughts?

I do think it would be a good idea to add more faults and fixes, or improve the few example ones I added, but this PR is mostly to get the code parts in.